include/nlohmann/detail/output/binary_writer.hpp

```diff
@@ -1090,15 +1090,14 @@ class binary_writer
             write_bson_entry_header(name, 0x10 /* int32 */);
             write_number<std::int32_t>(static_cast<std::int32_t>(j.m_data.m_value.number_unsigned), true);
         }
-        else if (j.m_data.m_value.number_unsigned <= static_cast<std::uint64_t>((std::numeric_limits<std::int64_t>::max)()))
+        else if (j.m_data.m_value.number_unsigned <= std::numeric_limits<std::uint64_t>::max())
```

Comment:
The call to `max` needs to be put in parentheses to work with MSVC. This is the reason the Windows jobs are currently failing.